.github/workflows/periodic_update.yml

```diff
@@ -0,0 +1,65 @@
+name: Periodic update
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API, or on a schedule.
+on:
+  workflow_dispatch:
+  schedule:
+  # At 10:32 on every day-of-month from 1 through 7 and on Wednesday.
```

Comment:
Can you translate this to plain English? It means first Wednesday of the month, no?